Gemfile

```diff
@@ -50,12 +50,17 @@ gem 'bootsnap', require: false
 
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/206868762-eb27958f-00cb-4fe9-82e5-56c6f9d9b64a.PNG)

- [ ] It's great that you have a license file in your project but it looks like a template 🤔 Kindly add your **name**, **current year** and **app-type** 😊 to make your project licensed to you 👍